kotlin/src/test/TestMethods.kt

```
@@ -549,12 +549,22 @@ class TestMethods {
                 // Now that Transport.kt uses GSon, this null property will be stripped from the request payload
                 hidden_value_domain_whitelist=null
             )
-            val actual = sdk.ok<UserAttribute>(sdk.create_user_attribute(body))
+            val groups = sdk.ok<Array<Group>>(sdk.all_groups("id, name"))
+            val group = groups.first()
+
+            val all = sdk.ok<Array<UserAttribute>>(sdk.all_user_attributes("id, name"))
+            var attribute = all.find { a -> a.name == body.name}
+            if (attribute == null) {
+                attribute = sdk.ok<UserAttribute>(sdk.create_user_attribute(body))
+            }
+            val attrGroup = UserAttributeGroupValue(group_id=group.id!!, user_attribute_id=attribute.id!!, value_is_hidden=false, value="1.2.3.4")
+            sdk.ok<Array<UserAttributeGroupValue>>(sdk.set_user_attribute_group_values(attribute.id!!, arrayOf(attrGroup))).first()
```

Are we expecting an error? If we are, should we throw an error here